frontend/src/feature/cards/components/CardDetailsComponent.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <v-card
+    class="pa-2 ma-2 mx-auto d-flex flex-column justify-space-between" max-width="500px" min-height="300px">
+    <v-card-title class="text-center">
+      {{ card.title }}
```

Comment:
iw ould drop class text-center and insert v-card-title with buttons, like this (replacing texts of course...):
```html
    <v-card-title class="d-flex justify-space-between align-center">
      <h4>{{ category.name }}</h4>
      <div class="mr-n6">
        <edit-mdi-button tooltip-text="Rename Category"
                         :disabled="!putUri"
                         :clickHandler="editCategory"/>
        <delete-mdi-button tooltip-text="Delete Category"
                           :disabled="!deleteUri"
                           :clickHandler="deleteCategory"/>
      </div>
    </v-card-title>
``´